README.md

```diff
@@ -27,10 +27,15 @@ repositories {
 }
 
 dependencies {
-    'com.basecamp:turbolinks:1.0.2'
+    compile 'com.basecamp:turbolinks:1.0.2'
 }
 ```
 
+Add the INTERNET permission to `AndroidManifest.xml`
```

Comment:
I don't think this section is necessary. While it's strictly true, it's not really the responsibility of the library's docs to inform an Android app's general permissions.